anesthetic/plot.py

```diff
@@ -327,7 +327,7 @@ def kde_plot_1d(ax, data, *args, **kwargs):
     kde = gaussian_kde(x, weights=w)
     p = kde(x)
     p /= p.max()
-    i = ((x < quantile(x, 0.999, w)) & (x > quantile(x, 0.001, w))) | (p > 0.1)
+    i = (p > 0)
```

Comment:
Was there an important reason why the data excluded the outer 0.2% ? If yes, I could bring this back in, but without an option of manually adjusting this, this is not ideal...

Reply:
I'm not 100% certain, but I'm pretty sure that this was associated with fastkde, which is now effectively deprecated with all the other changes that have been made since then. Does removing this have any obvious bad effects (either with or without fastkde)?

Reply:
It does seem to have been copied from fastkde. Removing it causes no obvious problems for kde, so I'd say removing it would be nice as it gets rid of some hard coded numbers that hinder customisation.